coremltools/converters/mil/frontend/torch/ops.py

```diff
@@ -4795,3 +4795,27 @@ def remainder(context, node):
     context.add(scaled_div)
     remainder_node = mb.sub(x=dividend, y=scaled_div, name=node.name)
     context.add(remainder_node)
+
+
+@register_torch_op
+def hann_window(context, node):
+    inputs = _get_inputs(context, node, expected=5)
+    # TODO: Support unknown window length
+    # TODO: Support unknown periodic
+    size = (inputs[0].val,)
+    if inputs[0].val == 1:
+        one = mb.fill(shape=size, value=1.0, name=node.name)
+        context.add(one)
+        return
+
+    ones = mb.fill(shape=size, value=1.0)
```

Comment:
I want to support dynamic window_length if enable, but `ones = mb.fill(shape=inputs[0], value=1.0)` does not work.

Reply:
`shape[0]` equals `window_length`.
Does this answer your question?

Reply:
yes